components/org.wso2.carbon.appmgt.gateway/src/main/java/org/wso2/carbon/appmgt/gateway/utils/RESTAPIAdminClient.java

```diff
@@ -0,0 +1,317 @@
+/**
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wso2.carbon.appmgt.gateway.utils;
+
+import org.apache.axis2.AxisFault;
+import org.wso2.carbon.appmgt.impl.utils.AbstractAPIGatewayAdminClient;
+import org.wso2.carbon.rest.api.stub.RestApiAdminAPIException;
+import org.wso2.carbon.rest.api.stub.RestApiAdminStub;
+import org.wso2.carbon.rest.api.stub.types.carbon.APIData;
+import org.wso2.carbon.appmgt.api.AppManagementException;
+
+import java.rmi.RemoteException;
+
+public class RESTAPIAdminClient extends AbstractAPIGatewayAdminClient {
```

Comment:
Formatting issues and Java doc comment issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files